server/src/main/scala/za/co/absa/atum/server/api/http/HttpMetrics.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.api.http
+
+import sttp.tapir.server.metrics.prometheus.PrometheusMetrics
+
+object HttpMetrics {
+
+  val prometheusMetrics: PrometheusMetrics[HttpEnv.F] = PrometheusMetrics[HttpEnv.F]("atum")
```

Comment:
I guess that this `"atum"` here is a Prometheus namespace? Would it be a good idea to have it split between environments, i.e. `atum-dev` etc? I don't remember how it looks like and I don't know how it's stored, I'm sorry it's possible that my question is not relevant :D

Reply:
You've raised a good point. Let me actually investigate if we could set the prefix (namespace) in Prometheus' configuration instead of in the application.

Reply:
So we could define labels which would identify metrics for a given environment etc. in the Prometheus.yml file.
```
scrape_configs:
  - job_name: 'my_application'
    static_configs:
      - targets: ['<your_application_address>']
        labels:
          environment: 'production'
          app: 'my_application'
```
How about we set environment label to 'dev/uat/prod' and app label to 'atum' and the prefix on line 23 that currently says "atum" we can change to "http" or keep its default value "tapir".?

Reply:
It appears in the metric name. Compare, the current one with "atum".
```
atum_request_duration_seconds_sum{method="POST",path="/api/v1/createPartitioning",phase="body",status="2xx"} 0.372
```

With the one using "http".
```
http_request_duration_seconds_sum{method="POST",path="/api/v1/createPartitioning",phase="body",status="2xx"} 0.372
```

If we use the labels for application and environment it would look like this.

```
http_request_duration_seconds_sum{app="atum",env="dev",method="POST",path="/api/v1/createPartitioning",phase="body",status="2xx"} 0.372
```

Similarly for the JVM/ZIO metrics.

```
jvm_memory_pool_bytes_used{app="atum",env="dev",pool="G1 Survivor Space",} 5769904.0 17102351351
```

I should say we can't change the name of those metrics from zio and jvm so the labels are the way to go I think.